src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -18,6 +18,7 @@ public static class MetricEventNames
 
         // language worker level events
         public const string WorkerInitializeLatency = "host.startup.outofproc.{0}worker.initialize.attempt{1}.latency";
+        public const string WorkerRuntimeLanguage = "host.startup.language.{0}";
```

Comment:
nit: HostStartupLanguage, to match naming convention for other host startup metrics. Also, move this up next to its brothers above (host.startup.latency, etc.)

Reply:
For accuracy, the metric name should also be `host.startup.runtime`